rootfs/scheduler/__init__.py

```diff
@@ -231,14 +232,15 @@
   "spec": {
```

Comment:
This is now really just a stub service that gets create that we update later (during deploy)